### DISCUSSION TITLE
Appearance customization.

### DISCUSSION THREAD

**Discussion by FairWindCo** [otto-torino/django-baton#244] [General]
Tell me if the possibility of customizing the appearance of the interface (fonts, colors) without rebuilding the script baton.min.js?

**Comment by abidibo** [comment]
Hi @FairWindCo , the only other method to customize the UI is to override all the styles you need, but this will be tedious.

**Reply by krystofbe** [reply]
Customizing the appearance is absolutely achievable, provided that one is well-versed in SCSS and understands Bootstrap internals thoroughly. Btw thanks for you great work with django-baton @abidibo 

We've actually created an internal fork of django-baton, which we've restyled to meet our needs. Here’s a sneak peek of how it looks:

<img width="1692" alt="grafik" src="https://github.com/otto-torino/django-baton/assets/12120657/69505b5f-2204-4185-a40a-7b7daa6f5ff7">
